pom.xml

```diff
@@ -412,11 +412,6 @@
                         </dependency>
                     </dependencies>
                 </plugin>
-                <plugin>
-                    <groupId>org.apache.maven.plugins</groupId>
-                    <artifactId>maven-deploy-plugin</artifactId>
```

Comment:
❓  How does `deploy`, `clean`, `install` work again? We don't need to explicitly declare it since the latest will be used?

Reply:
most of the core maven plugins are already defined as part of the maven super-pom. so unless there is some specific reason to fix them to a specific version they don't really need to be included in pluginManagement